How We Operate updates (#39)
data-lessons/instructor-training: _episodes/21-carpentries.md

their projects to form a new project called
[The Carpentries]({{ site.carpentries_site }})
under the fiscal sponsorship of
[Community Initiatives][CI].
Within this new organization structure, Software Carpentry and Data Carpentry retained their individual identities as Lesson Programs of The Carpentries.
On November 1, 2018, The Carpentries Executive Council
approved [Library Carpentry]({{ site.lc_site }})
as the third official Lesson Program of The Carpentries.

> The Carpentries core operations are run by a small Core Team, but the heart of the project consists of our global and local communities. Global Carpentries communities consist of Instructors, Instructor Trainers,

Reviewer: Include link to the core team page?

are all trademarked.
You may only call a workshop a Data Carpentry, Library Carpentry, or Software Carpentry workshop if:

*   it covers the core curriculum for that lesson program (see below),
*   at least one of the instructors is a certified Carpentries Instructor,
*   you run our standardized pre- and post-workshop assessments and ensure everyone participates.

Note that as long as you have at least one certified Instructor, the other
instructors and helpers at a branded workshop do not have to be certified. We
rely on your judgement as a certified Instructor to recruit and orient volunteers who
will support you in implementing The Carpentries ethos and practices. **We do expect you to have at least one co-Instructor and helpers**, even if they are not 
certified instructors.

> ## Local Support
>

> > There is a great list here in the handbook [Helper Checklist and Suggestions][docs-helper-checklist].

Reviewer 2: ```suggestion
> There is a helpful list for planning here in the handbook [Helper Checklist and Suggestions][docs-helper-checklist].
```

making this more specific

It is only when you want to run a branded Carpentries workshop using the materials
that the requirements listed above are needed. Other workshops may be described as being "Carpentries-based", using Carpentries materials, etc.

### What is the Core Curriculum?

For Data Carpentry:
* a Data Carpentry [lesson][dc-lessons]
on data organization and 
* three
other modules in the same domain from the Data Carpentry curriculum
Published curricula include the Ecology, Genomics, Social Science and Geospatial workshop materials. Other domains are in development, including Image 
Processing, Economics, and Astronomy.

For Library Carpentry:
* three to four of the
[core lessons][lc-lessons], which include an introduction to data, the UNIX shell, OpenRefine, and Git
Library Carpentry also maintains an "extended" set of lessons
that can be taught in addition to the core curriculum
These lessons are taught infrequently or are under development and include:
SQL, webscraping, Python and an introduction to data for archivists.

For Software Carpentry:
* [lessons][swc-lessons]
on version control (e.g. Git)
* the UNIX shell,
and 
* a programming language (e.g. R or Python)

> Within these guidelines, there is flexibility in which episodes of the lesson you cover,

Reviewer: Do we explain the terminology and structure of episode vs. lesson anywhere?

Reviewer 2: i think it's only implied

Reviewer 2: Could we add an exercise at this point to:

> in small groups find the term that fits each of the following definitions:

where the solutions are:
- lesson
- episode
- workshop
- lesson program

Author: I added something to this effect -- I broadened it out a little to hit another common misconception and focus on the problem of Carpentries jargon more broadly.

which exercises you use, and whether you include optional materials (e.g. callouts) and
optional episodes.

### Instructor Badges Apply Universally

> Certified Carpentries Instructors can teach curricula for any of the lesson programs

Reviewer: We are using the terminology of both certification and badges. That could be confusing. Recommendation to either explain or stick to one type of language vs. another.

Author: I think this jargon issue is maybe broader than can be addressed here. I'd rather not take a stand on one or the other at the moment. There is a chance that we may develop other badging opportunities 🤞 , in which case we would want a systemic review of badging vs. certification.

Author: I did change the header to use Certification, in case that helps!

information, see the description of [the Instructor checkout procedure]({{ page.root }}/checkout/).

## How to Run a Carpentries Workshop

[**The Carpentries Handbook**][docs] is a community-developed resource that
provides tips, checklists, and points of contact for nearly all Carpentries-related activities in one location.
The Carpentries Handbook is the definitive source for policies and information.
Here, we will cover some information about actually running a workshop.

Briefly, there are two types of Carpentries workshops: **centrally-organized** and
**self-organized**. Centrally-organized workshops include support from
[Regional Coordinators][rcs-page] and our workshop administration team.
The Carpentries team will:
- recruit Instructors to teach
- manage workshop registration
- coordinate communication between Instructors and Hosts
- provide links to pre- and post- workshop survey data to Instructors and Hosts. (Note that the links to the surveys themselves are generated by the workshop website)

Centrally organized workshops *do not* include:
- travel arrangements or expense management

> - recruitment of helpers (we recommend 1 per 8-10 learners)

Reviewer: Can we consider putting the helper ratio recommendation up where we talk about helpers too?

- creation of a workshop website
- communication with learners before or after a workshop
- local logistics (e.g. catering)

> These must be arranged by the Host. Local instructors will be prioritized for selection, and constraints on recruitment may also be noted in the request form. 

Reviewer: What does "constraints on recruitment" mean?

Author: ```suggestion
These must be arranged by the Host. Local instructors will be prioritized for selection, and constraints on recruitment (e.g. maximum travel distance) may also be noted in the request form. 
```

For self-organized workshops, the host institution is responsible for all logistics and administration. However, if a self-organized workshop request form is 
submitted, The Carpentries will send links to the pre- and post- workshop survey response data to all identified Instructors and Hosts. You can also ask us, or 

> reach out on our community channels, for advice! 

Reviewer: Technically couldn't you reach out to community channels for advice no matter who is organizing the workshop?

> To request a centrally-organised workshop, register a self-organised workshop, or ask for more information on either, your [start page is here][workshops-form]. Be sure to submit your request in time to make use of those pre-workshop survey data!

Reviewer: Not completely clear what "in time to make use of your pre-workshop survey data" means. Can we make our recommendation on timelines clearer?

The Carpentries Handbook is the definitive source for policies and information.
Here, we will cover some information about actually running a workshop.

Briefly, there are two types of Carpentries workshops: **centrally-organized** and
**self-organized**. Centrally-organized workshops include support from
[Regional Coordinators](https://carpentries.org/regionalcoordinators/).
Regional Coordinators will:
- recruit Instructors to teach
- manage workshop registration
- coordinate communication between Instructors and Hosts

> - provide links to pre- and post- workshop survey data to Instructors and Hosts. (Note that the links to the surveys themselves are generated by the workshop website)

Reviewer 3: technically Talisha does this bit

Author: Thanks -- good point! I think this is accurate now?

Centrally organized workshops *do not* include:
- travel arrangements or expense management
- recruitment of helpers (we recommend 1 per 8-10 learners)
- creation of a workshop website
- communication with learners before or after a workshop
- local logistics (e.g. catering)

These must be arranged by the Host. Local instructors will be prioritized for selection, and constraints on recruitment may also be noted in the request form. 

> For self-organized workshops, the host institution is responsible for all logistics and administration. However, if a self-organized workshop request form is submitted, The Carpentries will send links to the pre- and post- workshop survey response data to all identified Instructors and Hosts. You can also ask us, or reach out on our community channels, for 

Reviewer 3: Should we add a link to the WRF here?

>
> At The Carpentries, we know all too well that tools and technical procedures can present a serious barrier to progress. 
> That is why we exist! So we appreciate the irony when participation in our community demands tools or technical 
> procedures that may not be familiar or accessible to all who wish to contribute. GitHub, in particular is often recognized as an obstacle;
> this is something our community has [discussed extensively][git-blog].
>
> We are actively invested in making our processes as friendly and inclusive as possible, and that includes supporting our 
> community in surmounting the barriers presented by our own technologies of choice. Watch our channels for 
> opportunities to develop your skills as a contributor! In the mean time, if you
> have any ideas about how we can make our resources more contributor-friendly,
> please [let us know][contact-page].
{: .callout}

### Community Roles

The Carpentries is a complex, global community teaching workshops and developing lessons are only two examples of many ways to participate! 
Whatever your interests or strengths, we hope you will find a place where you can grow with us. 

> ## Who Does What
> 

> > Have a look at this form to match different Carpentries roles with short descriptions. If you are not sure about some of them or if you want to know more,

Reviewer 2: where is the form?

Author: Having given further thought to this, I have determined that a handout is not necessary and it would be desirable to eliminate the associated maintenance load. New activity reads:

> Select one role from the list below that interests you. Using the the descriptions on [The Carpentries community website][community-page], write
> 1) a short definition of the role and 2) a question that you have (or that you imagine someone else might have) about the role. Are there roles you
> would like to see that are not listed? Note that, too!

Reviewer 2: extra love for less maintainence!

> have a look at the descriptions on [The Carpentries community website][community-page]. 
> Working on your own, match up the roles with the descriptions. 
>
>> ## Solution
>> 1. Executive Council
>> 2. Mentors
>> 3. Trainers
>> 4. Instructors
>> 5. Lesson Developers
>> 6. Code of Conduct Committee
>> 7. Instructor Development Committee
>> 8. Community Facilitators
> {: .solution}
> This exercise should take about 10 minutes.
{: .challenge}

### Communications!
There are a dizzying number of ways to keep in touch with The Carpentries communities. It is not necessary to follow all of them! Everyone has their own favorite
ways to keep in touch, and we do our best to make that possible. 

Want to **listen**? 
- Sign up for our newsletter
- Follow us on Twitter, Facebook, or LinkedIn

Want to **interact** (or listen with options to engage)?
- Join our Slack organisation
- Join our Email lists (start with "Discuss"!)

Want to **talk** (or listen in on meetings)?
- Sign up for Community Discussions (or just drop in if there is space!) or other events when announced
- Explore taking on one of the Roles identified above

> ## Get Connected
>

> > Take a couple of minutes to sign up for The Carpentries [channels][connect-page] you want to stay involved with.
> {: .challenge}

Reviewer 2: ```suggestion
> Take a couple of minutes to sign up for The Carpentries [channels][connect-page] you want to stay involved with.
> When you're done share what channel you're most excited about on the etherpad.
{: .challenge}
```

I usually use this prompt so that I know when trainees are done,I figure it's worth sharing.
